test/lib/build.test.js

```diff
@@ -1597,11 +1597,11 @@ describe('Build Model', () => {
                 state: 'ENABLED',
                 archived: false,
                 pipeline: Promise.resolve(pipelineMockB),
-                permutations: [{ annotations: { 'beta.screwdriver.cd/executor:': 'k8s-vm' }, provider }],
+                permutations: [{ annotations: { 'beta.screwdriver.cd/executor:': 'k8s-test' }, provider }],
```

Comment:
do we have an executor named `k8s-test`?

Reply:
No, we don't have the `k8s-test` executor.
However, this executor name is only used as a test string and does not have any dependencies.

Here, we are checking that the annotation is passed correctly, so I think using a test string is better since we can clearly see this string being set up from the test code.
(If we use `k8s`, then it might be set up from the default executor setting.)